HST Office: module names in file type and filter display names.

The displayed names of file types and filters that name a module follow the application
names: Calc -> Sheets, Impress -> Slides ("Calc 8" -> "Sheets 8", "HTML Document (Calc)" ->
"HTML Document (Sheets)", "Spreadsheets (Calc)" -> "Spreadsheets (Sheets)" ...). Only the
UIName / DisplayName texts change: filter names, type names, extensions and service names are
API and macro identifiers and stay as they are. The Vietnamese translations follow through
l10n/vi-overrides.jsonl (msgid_new).

--- a/filter/source/config/fragments/filters/impress_StarOffice_XML_Draw.xcu
+++ b/filter/source/config/fragments/filters/impress_StarOffice_XML_Draw.xcu
@@ -25,6 +25,6 @@
         <prop oor:name="TemplateName"/>
         <prop oor:name="DocumentService"><value>com.sun.star.presentation.PresentationDocument</value></prop>
         <prop oor:name="UIName">
-            <value xml:lang="en-US">OpenOffice.org 1.0 Drawing (Impress)</value>
+            <value xml:lang="en-US">OpenOffice.org 1.0 Drawing (Slides)</value>
         </prop>
     </node>
--- a/filter/source/config/fragments/filters/calc_HTML_WebQuery.xcu
+++ b/filter/source/config/fragments/filters/calc_HTML_WebQuery.xcu
@@ -25,6 +25,6 @@
         <prop oor:name="TemplateName"/>
         <prop oor:name="DocumentService"><value>com.sun.star.sheet.SpreadsheetDocument</value></prop>
         <prop oor:name="UIName">
-            <value xml:lang="en-US">Web Page Query (Calc)</value>
+            <value xml:lang="en-US">Web Page Query (Sheets)</value>
         </prop>
     </node>
--- a/filter/source/config/fragments/filters/HTML__StarCalc_.xcu
+++ b/filter/source/config/fragments/filters/HTML__StarCalc_.xcu
@@ -25,6 +25,6 @@
         <prop oor:name="TemplateName"/>
         <prop oor:name="DocumentService"><value>com.sun.star.sheet.SpreadsheetDocument</value></prop>
         <prop oor:name="UIName">
-            <value xml:lang="en-US">HTML Document (Calc)</value>
+            <value xml:lang="en-US">HTML Document (Sheets)</value>
         </prop>
     </node>
--- a/filter/source/config/fragments/filters/calc_xml_Orcus.xcu
+++ b/filter/source/config/fragments/filters/calc_xml_Orcus.xcu
@@ -14,6 +14,6 @@
     <prop oor:name="TemplateName"/>
     <prop oor:name="DocumentService"><value>com.sun.star.sheet.SpreadsheetDocument</value></prop>
     <prop oor:name="UIName">
-        <value xml:lang="en-US">XML Document (Calc)</value>
+        <value xml:lang="en-US">XML Document (Sheets)</value>
     </prop>
 </node>
--- a/filter/source/config/fragments/filters/Rich_Text_Format__StarCalc_.xcu
+++ b/filter/source/config/fragments/filters/Rich_Text_Format__StarCalc_.xcu
@@ -21,7 +21,7 @@
         <prop oor:name="FilterService"/>
         <prop oor:name="UserData"><value></value></prop>
         <prop oor:name="UIName">
-            <value xml:lang="en-US">Rich Text Format (Calc)</value>
+            <value xml:lang="en-US">Rich Text Format (Sheets)</value>
         </prop>
         <prop oor:name="FileFormatVersion"><value>0</value></prop>
         <prop oor:name="Type"><value>writer_Rich_Text_Format</value></prop>
--- a/filter/source/config/fragments/filters/impress8_draw.xcu
+++ b/filter/source/config/fragments/filters/impress8_draw.xcu
@@ -25,6 +25,6 @@
         <prop oor:name="TemplateName"/>
         <prop oor:name="DocumentService"><value>com.sun.star.presentation.PresentationDocument</value></prop>
         <prop oor:name="UIName">
-            <value xml:lang="en-US">ODF Drawing (Impress)</value>
+            <value xml:lang="en-US">ODF Drawing (Slides)</value>
                     </prop>
     </node>
--- a/filter/source/config/fragments/filters/impress_html_Export.xcu
+++ b/filter/source/config/fragments/filters/impress_html_Export.xcu
@@ -25,6 +25,6 @@
         <prop oor:name="TemplateName"/>
         <prop oor:name="DocumentService"><value>com.sun.star.presentation.PresentationDocument</value></prop>
         <prop oor:name="UIName">
-            <value xml:lang="en-US">HTML Document (Impress)</value>
+            <value xml:lang="en-US">HTML Document (Slides)</value>
         </prop>
     </node>
--- a/filter/source/config/fragments/filters/calc_json_Orcus.xcu
+++ b/filter/source/config/fragments/filters/calc_json_Orcus.xcu
@@ -14,6 +14,6 @@
     <prop oor:name="TemplateName"/>
     <prop oor:name="DocumentService"><value>com.sun.star.sheet.SpreadsheetDocument</value></prop>
     <prop oor:name="UIName">
-        <value xml:lang="en-US">JSON Document (Calc)</value>
+        <value xml:lang="en-US">JSON Document (Sheets)</value>
     </prop>
 </node>
--- a/filter/source/config/fragments/types/calc8_template.xcu
+++ b/filter/source/config/fragments/types/calc8_template.xcu
@@ -23,7 +23,7 @@
         <prop oor:name="Preferred"><value>false</value></prop>
         <prop oor:name="PreferredFilter"><value>calc8_template</value></prop>
         <prop oor:name="UIName">
-            <value xml:lang="en-US">Calc 8 Template</value>
+            <value xml:lang="en-US">Sheets 8 Template</value>
         </prop>
-        <prop oor:name="ClipboardFormat"><value>Calc 8 Template</value></prop>
+        <prop oor:name="ClipboardFormat"><value>Sheets 8 Template</value></prop>
     </node>
--- a/filter/source/config/fragments/types/calc8.xcu
+++ b/filter/source/config/fragments/types/calc8.xcu
@@ -23,7 +23,7 @@
         <prop oor:name="Preferred"><value>true</value></prop>
         <prop oor:name="PreferredFilter"><value>calc8</value></prop>
         <prop oor:name="UIName">
-            <value xml:lang="en-US">Calc 8</value>
+            <value xml:lang="en-US">Sheets 8</value>
         </prop>
-        <prop oor:name="ClipboardFormat"><value>Calc 8</value></prop>
+        <prop oor:name="ClipboardFormat"><value>Sheets 8</value></prop>
     </node>
--- a/filter/source/config/fragments/types/impress8.xcu
+++ b/filter/source/config/fragments/types/impress8.xcu
@@ -23,7 +23,7 @@
         <prop oor:name="Preferred"><value>true</value></prop>
         <prop oor:name="PreferredFilter"><value>impress8</value></prop>
         <prop oor:name="UIName">
-            <value xml:lang="en-US">Impress 8</value>
+            <value xml:lang="en-US">Slides 8</value>
         </prop>
-        <prop oor:name="ClipboardFormat"><value>Impress 8</value></prop>
+        <prop oor:name="ClipboardFormat"><value>Slides 8</value></prop>
     </node>
--- a/filter/source/config/fragments/types/impress8_template.xcu
+++ b/filter/source/config/fragments/types/impress8_template.xcu
@@ -23,7 +23,7 @@
         <prop oor:name="Preferred"><value>false</value></prop>
         <prop oor:name="PreferredFilter"><value>impress8_template</value></prop>
         <prop oor:name="UIName">
-            <value xml:lang="en-US">Impress 8 Template</value>
+            <value xml:lang="en-US">Slides 8 Template</value>
         </prop>
-        <prop oor:name="ClipboardFormat"><value>Impress 8 Template</value></prop>
+        <prop oor:name="ClipboardFormat"><value>Slides 8 Template</value></prop>
     </node>
--- a/officecfg/registry/data/org/openoffice/Office/UI.xcu
+++ b/officecfg/registry/data/org/openoffice/Office/UI.xcu
@@ -31,7 +31,7 @@
         </node>
         <node oor:name="com.sun.star.sheet.SpreadsheetDocument" oor:op="replace">
           <prop oor:name="DisplayName">
-            <value xml:lang="en-US">Spreadsheets (Calc)</value>
+            <value xml:lang="en-US">Spreadsheets (Sheets)</value>
           </prop>
           <prop oor:name="Filters">
             <value oor:separator=";">ADO Rowset XML;Calc MS Excel 2007 Binary;Calc MS Excel 2007 XML;Calc MS Excel 2007 XML Template;calc_StarOffice_XML_Calc_Template;calc8;calc8_template;chart8;DIF;Lotus;MiniCalc (Palm);MS Excel 2003 XML;MS Excel 97;MS Excel 97 Vorlage/Template;OpenDocument Spreadsheet Flat XML;Pocket Excel;Quattro Pro 6.0;StarOffice XML (Calc);StarOffice XML (Chart);Text - txt - csv (StarCalc)</value>
@@ -39,7 +39,7 @@
         </node>
         <node oor:name="com.sun.star.presentation.PresentationDocument" oor:op="replace">
           <prop oor:name="DisplayName">
-            <value xml:lang="en-US">Presentations (Impress)</value>
+            <value xml:lang="en-US">Presentations (Slides)</value>
           </prop>
           <prop oor:name="Filters">
             <value oor:separator=";">Impress MS PowerPoint 2007 XML;Impress MS PowerPoint 2007 XML AutoPlay;Impress MS PowerPoint 2007 XML Template;impress_StarOffice_XML_Draw;impress_StarOffice_XML_Impress_Template;impress8;impress8_template;MS PowerPoint 97;MS PowerPoint 97 AutoPlay;MS PowerPoint 97 Vorlage;OpenDocument Presentation Flat XML</value>
